.pre-commit-config.yaml

```diff
@@ -66,16 +70,6 @@ repos:
             --linelength=88,
             '--filter=-readability/casting,-runtime/int,-build/include_subdir,-readability/fn_size'
         ]
--   repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
-    hooks:
-    -   id: flake8
-        # Need to patch os.remove rule in pandas-dev-flaker
-        exclude: ^ci/fix_wheels.py
-        additional_dependencies: &flake8_dependencies
-        - flake8==6.0.0
-        - flake8-bugbear==22.7.1
-        - pandas-dev-flaker==0.5.0
```

Comment:
Completely removing flake8 here means we also stop using pandas-dev-flaker. I am not familiar with it, but are there things we would like to keep from that? Is it possible to run flake8 _only_ with the pandas-dev-flaker checks?

Reply:
I looked into that but it's not so simple...I think the simplest thing would be to revert the pandas-dev-flaker PR https://github.com/pandas-dev/pandas/pull/50519

Reply:
OK, I see, that's indeed also a good option.